clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -38,9 +45,11 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
     return () => clearTimeout(delayBanner);
   }, [setIsShown, waitBeforeShow]);
 
-  const navigateToPrivacyCenter = (): void => {
-    if (privacyCenterUrl) {
+  const handleManagePreferencesClick = (): void => {
+    if (experience.component === ExperienceComponent.PRIVACY_CENTER) {
       window.location.assign(privacyCenterUrl);
+    } else {
+      onOpenModal();
     }
   };
```

Comment:
❗ I'm really not sure if this is the right logic. I thought we sometimes wanted to go to PC and sometimes wanted to open the modal here, but I think we might _only_ want to open the modal here? and leave PC opening to links, is that right?